configuration.md

```diff
@@ -0,0 +1,104 @@
+# Proposal for configuring supported
+
+Teams may want to customize the parameters of a support policy. For instance, a team may want to ignore certain dependencies, i.e. not enforce upgrades for certain dependencies. They may also want different upgrade cadences for specific dependencies. Lastly, in many cases, simply turning on a support policy and expecting consumers to immediately adhere to the policy is untenable (for instance, a major upgrade with a migration may be emminently required shortly after turning on a support policy). There needs to be a way to provide a grace period for onboarding to the support policy, while still providing a warning for when dependencies must be updated.
+
+We will not require a configuration file, and will default to 4, 2, and 1 quarter upgrade periods for major, minor, and patch versions respectively.
+
+```JSON
```

Comment:
where this JSON will reside? 
One place I think will be the root of the project being evaluated. 
Are we planning to provide CI options to achieve the same?

Reply:
Good question, I don't have a strong opinion here besides ensuring that both the programatic API and CI tool can specify a custom JSON file.

Reply:
I think we should have both options. Most of the MP will use the JSON file if there are certain deps to be ignored all the time or list is too huge.